snyk-monitor/templates/configmap.yaml

```diff
@@ -3,9 +3,35 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-excluded-namespaces
+  labels:
+    app.kubernetes.io/name: {{ include "snyk-monitor.name" . }}
+    helm.sh/chart: {{ include "snyk-monitor.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
   excludedNamespaces: |-
     {{- range .Values.excludedNamespaces }}
     {{ . }}
     {{- end }}
 {{ end }}
+---
```

Comment:
Do we need `{{ if .Values.policyOrgs }}`? Or we want this policy be set for every kubernete-monitor by default?

Reply:
We want it set by default to make it easier for on-boarding